samples/angular/scripts/disconnected-mode-proxy.ts

```diff
@@ -25,6 +25,8 @@ const proxyOptions = {
   appName: config.appName,
   watchPaths: ['./data'],
   language: config.language,
+  // Additional transpilation is not needed
+  requireArg: '',
```

Comment:
This makes sense, but would it be cleaner to pass in  `'none'` 
or something other than an empty string, and handle it inside of the `ManifestManager`?

Something like the following to go [here](https://github.com/Sitecore/jss/blob/19954b1e5a82e0f267fbfb3434f12c3271d8359e/packages/sitecore-jss-dev-tools/src/manifest-manager.ts#L72)?
```
if (requireArg === 'none' || requireArg === false) {
    this.requireArg = '';
}
```

Reply:
Good idea, resolved